data/mods/PKs_Rebalance/pk_terrain.json

```diff
@@ -0,0 +1,108 @@
+[
+    {
+        "type" : "terrain",
+        "id" : "t_guardrail_bg_dp",
+        "edit-mode": "modify",
+        "name": "guard rail",
+        "symbol": "#",
+        "color": "ltgray",
+        "move_cost": 3,
+        "flags": ["TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THINOBSTACLE", "ROAD"],
```

Comment:
I'm going to spend some time tomorrow for reading more info about your recent activity (related to lint & json files), so I'll let you know if I have something to add to your PRs.